### ISSUE TITLE
Decrease latency for speech recognition model

### ISSUE THREAD

**Issue by bharat-patidar** [tensorflow/tensor2tensor#1813]
Hi,
I am working on a pre-trained t2t speech recognition model and trying to reduce its latency of model inference on CPU machine.
What all could be the possible ways to decrease decoding time/latency without any significant change in accuracy? Can we use tflite for the same?
